hibike/spawn_virtual_devices.py

```diff
@@ -0,0 +1,38 @@
+import csv
```

Comment:
Not reviewing this file now, pending resolving the issues raised in the last PR.